components/Contact.vue

```diff
@@ -1,28 +1,37 @@
 <template>
-  <div class="fixed left-0 w-screen h-full flex items-center justify-center">
-    <img class="absolute bottom-0 right-0" src="../static/contact/blackblob.png">
-    <img class="absolute bottom-0 left-0 select-none" src="../static/contact/greenblob.png">
-    <div class="mx-24 w-11/12 z-20">
-      <div class="font-bold text-6xl mb-10">Get in touch</div>
-      <div class="rounded-3xl bg-app-gray-2 flex py-8">
-        <div class="relative rounded-3xl w-1/3">
-          <img class="absolute left-0 bottom-0 object-cover transform origin-bottom-left scale-125" src="../static/contact/bunchofleafs.png">
-          <img class="absolute bottom-0 object-cover" src="../static/contact/unboggled.png">
+  <div class="w-full flex justify-center">
+    <div class="relative mx-auto my-16 w-11/12 z-20">
+      <div class="relative font-bold z-30 text-6xl mb-10 text-center lg:text-left">Get in touch</div>
+      <div class="absolute top-0 w-full flex justify-center">
+      <img draggable=false class="transform -translate-y-24 sm:-translate-y-40 translate-x-0 lg:translate-x-48" src="../static/contact/leaves.png">
+      </div>
+      <div class="relative rounded-3xl bg-app-gray-2 flex flex-col-reverse lg:flex-row justify-center lg:justify-end items-center">
+        <div class="relative lg:absolute lg:left-0 w-full h-full rounded-3xl flex items-end justify-center lg:justify-start select-none">
+          <img 
+            draggable=false 
+            class="hidden lg:block lg:absolute lg:left-0 object-cover z-10 w-1/3 h-full rounded-2xl" 
+            src="../static/contact/bunchofleaves.png"
+          >
+          <img 
+            draggable=false 
+            class="lg:pl-6 lg:pt-6 object-contain z-20 w-1/2 lg:w-1/4 max-h-full" 
+            src="../static/contact/unboggled.png"
+          >
         </div>
-        <form class="flex w-2/3 flex-col justify-center">
-          <div class="w-full flex flex-row justify-evenly items-center">
-            <div class="flex flex-col w-1/2 mx-8">
+        <form class="z-30 flex w-full lg:w-2/3 flex-col justify-center pt-8 pb-0 lg:pb-8" @submit="submit">
```

Comment:
I'd suggest either starting a generic form or making reusable input components, it'll trim down the amount of html required here

Reply:
Do you have any suggestions on how to do this? I don't think reusable input components would work, because there are 4 total input (2 `input`s, 1 `textarea`, and 1 submit `button`). The 2 input components differ slightly in CSS, and I'm not sure if it'd be worth it to construct a new reusable input component. Also, I'm not too sure what you mean by generic form.

Reply:
DW about this, just keep it how it is, I've been working on a side project for a very reusable generic form across any project and I'll implement it here.

Reply:
Generic form is just basically a reusable form that gets filled in from a array or object such as:
```
  <form :class="formClass" @submit.prevent="action()">
      <div v-for="(field, i) in fields" :key="i">
        <TextArea
          v-if="field.variant === 'textarea'"
          v-model="formData[field.name]"
          :name="field.name"
          :label="field.label"
          :placeholder="field.placeholder"
        />
        <MultiUpload v-if="field.variant === 'upload'" :name="field.name" />
        <Input
          v-else
          v-model="formData[field.name]"
          :name="field.name"
          :label="field.label"
          :placeholder="field.placeholder"
          :input-type="field.type"
        />
      </div>
      <button class="btn-orange mt-4" type="submit">Submit</button>
    </form>
```